spread.yaml

```diff
@@ -541,7 +543,9 @@ suites:
         restore: |
             #shellcheck source=tests/lib/reset.sh
             "$TESTSLIB"/reset.sh
-            if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
+            #shellcheck source=tests/lib/systems.sh
+            "$TESTSLIB"/systems.sh
```

Comment:
Is there a leading `.` missing, i.e.: `. "$TESTSLIB"/systems.sh`?

Reply:
Thanks @mvo for the review, change already done.